Modules/xASL_module_ASL.m

```diff
@@ -208,6 +208,10 @@
 
 if ~isfield(x.Q, 'bUseBasilQuantification') || isempty(x.Q.bUseBasilQuantification)
 	x.Q.bUseBasilQuantification = false;
+    
```

Comment:
Nice, but this goes wrong if the user accidentally specified `bUseBasilQuantification` as `false`, this would skip multiTE and multiPLD. In this case, there should be a warning to the user that we only have multi-TE or multi-PLD processing with BASIL.

Reply:
Yes - that warning of course is in the quantification module...

Reply:
Let's move all those warnings concerning WHAT the ASL module will run, with what booleans, to the start of the ASL module. This makes it clear to the user right at the start of a module, rather than in the middle of running something, don't you agree? You're right that this is quantification-specific, but in a way this concerns the whole ASL module. To the user, it doesn't matter which submodule or wrapper lacks functionality with a wrong setting.

Shall we create a new issue or move this to an existing issue:
* Ensure that important warnings for boolean/settings/sequence/flavors combinations are issued at the start of the ASL module rather than somewhere in the middle of the code